content/docs/components/serving/overview.md

```diff
@@ -196,14 +196,14 @@ For TensorFlow models you can use TensorFlow Serving for
 However, if you plan to use multiple frameworks, you should consider KFServing
 or Seldon Core as described above.
 
-## NVIDIA TensorRT Inference Server
+## NVIDIA Triton Inference Server
 
-NVIDIA TensorRT Inference Server is a REST and GRPC service for deep-learning
-inferencing of TensorRT, TensorFlow and Caffe2 models. The server is
+NVIDIA Triton Inference Server is a REST and GRPC service for deep-learning
```

Comment:
Can we add a line here mentioning that this was previously referred to as TensorRT. That would allow people who might be looking for TensorRT to relate to the fact that it has been renamed.

Reply:
Done